web/styles/portico/markdown.css

```diff
@@ -382,6 +382,10 @@
     }
 
     & pre {
+        /* Relative positioning ensures the copy-to-
+        clipboard button remains in the top-left corner */
+        position: relative;
```

Comment:
Does this have any side effects?

Reply:
No, I don't think so. From what I learned, relative positioning only changes an element if positioning constraints are specified.